### DISCUSSION TITLE
what to do with bodyless responses?

### DISCUSSION THREAD

**Discussion by saroar** [pointfreeco/swift-url-routing#49] [Q&A]
For example I want to delete user and get back HttpStatus ok 
get return from server(Vapor 4) this return HTTPStatus.ok

```
public struct UserClient {
    public let delete: @Sendable (String) async throws -> Void
    public init(
.....
        delete: @escaping @Sendable (String) async throws -> Void
    ) {
......
        self.delete = delete
    }
}

```
<img width="1233" alt="Screenshot_2022-09-07_at_10 59 00" src="https://user-images.githubusercontent.com/8770772/188824029-899bc68e-61cc-4df4-92dd-a0ed5243d41b.png">

I have solution but like to know about body less responses 
solution is create new struct and send back to client

**Comment by saroar** [comment]
do we have any news ? @stephencelis how I can receive body less request ? as I see it not bug may be its new features? or already have sone fun ? but I dont get any documents about this 
Thanks

**Comment by stephencelis** [comment]
@saroar If you don't need to decode a response, you can use `URLRoutingClient.data(for:)` instead of `decodedResponse(for:)`:

https://github.com/pointfreeco/swift-url-routing/blob/5195a70931de218af8a9fc8a1e07473bd5c8aa90/Sources/URLRouting/Client/Client.swift#L29-L36

**Reply by saroar** [reply]
<img width="753" alt="Screenshot 2022-11-24 at 12 26 35" src="https://user-images.githubusercontent.com/8770772/203747145-1476358f-91d3-4f1d-bfbc-0cda5f9db2b6.png">
